pkg/osvscanner/scan.go

```diff
@@ -240,3 +238,16 @@ func getRootDir(path string) string {
 
 	return ""
 }
+
+func determineOS() plugin.OS {
+	switch runtime.GOOS {
+	case "windows":
+		return plugin.OSWindows
+	case "darwin":
+		return plugin.OSMac
+	case "linux":
+		return plugin.OSLinux
+	default:
+		return plugin.OSAny
```

Comment:
I'm pretty sure its safe to return this rather than error, though (assuming we don't actually want to error) it could be nice if scalibr had a `plugin.OSUnknown` either as a new value or an alias for this one, since that's what we want to represent here

Reply:
Hmm, can this log a warning that OS "xyz" is not officially supported.